resource/resource_graph.go

```diff
@@ -119,40 +118,6 @@ func (g *Graph) Clone() *Graph {
 	return g.clone()
 }
 
-// ExportDot exports the resource graph as a DOT representation for visualization.
-// DOT reference: https://graphviz.org/doc/info/lang.html
-func (g *Graph) ExportDot() (string, error) {
```

Comment:
After I looked at this from 10,000 feet, I felt I added enough to move the implementation to a separate file. Certainly not my default to split up a type and its member functions into different files.